MalwareAnalyzer/views.py

```diff
@@ -85,6 +85,14 @@ def update_malware_db():
     """Check for update in malware DB"""
     try:
         url = "https://www.malwaredomainlist.com/mdlcsv.php"
+        if settings.UPSTREAM_PROXY_ENABLED:
+            if settings.UPSTREAM_PROXY_USERNAME is None :
+                proxy_host = settings.UPSTREAM_PROXY_IP + ':' + settings.UPSTREAM_PROXY_PORT
+            else:
+                proxy_host = settings.UPSTREAM_PROXY_USERNAME + ":" + settings.UPSTREAM_PROXY_PASSWORD + "@" + settings.UPSTREAM_PROXY_IP + ':' + settings.UPSTREAM_PROXY_PORT
+            proxy = urllib2.ProxyHandler({'https': proxy_host})
```

Comment:
Will `'https'` support http proxies as well?

Reply:
i can add both support (https + http )

Reply:
Yes, you can add an option in `settings.py` to choose between https or http.something like
UPSTREAM_PROXY_TYPE = "http"